src/main/java/edu/tamu/app/ProjectInitialization.java

```diff
@@ -31,8 +31,22 @@ public void run(String... args) throws Exception {
         remoteProjectManagerRepo.findAll().forEach(versionManagementSoftware -> {
             managementBeanRegistry.register(versionManagementSoftware);
         });
-        if (cardTypeRepo.findByIdentifier("Feature") == null) {
-            cardTypeRepo.create(new CardType("Feature", new HashSet<String>(Arrays.asList(new String[] { "Story" }))));
+        CardType type = cardTypeRepo.findByIdentifier("Feature");
```

Comment:
`Feature`, `Request`, `Issue`, & `Defect` should be constants. It is also curious that we are coding to values assumed to be in the repository.